tests/data/group_system/group_auditing/group_auditd_configure_rules/group_audit_privileged_commands/rule_audit_rules_privileged_commands_sudo/rhel7_augenrules_default.fail.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_ospp
+# remediation = bash
+
+# augenrules is default for rhel7
+# This is a trick to fail setup of this test in rhel6 systems
+ls  /usr/lib/systemd/system/auditd.service
```

Comment:
I know I myself introduced some "default" tests. But it has no value to actually run such scenario. Please remove